src/docs/sphinx/basicExamples/triaxialDriver/Example.rst

```diff
@@ -0,0 +1,263 @@
+This stress has a negative value due to the negative sign convention for compressive stress in GEOSX.
+
+
+Then, ``steps="200"`` defines the number of load steps and ``output="simulationResults.txt"`` specifies an output file to which the simulation results will be written. 
+
+
+.. literalinclude:: ../../../../../inputFiles/solidMechanics/triaxialDriver_ExtendedDruckerPrager.xml
+    :language: xml
+    :start-after: <!-- SPHINX_TASK -->
+    :end-before: <!-- SPHINX_TASK_END -->
+
+
+In addition to triaxial tests, volumetric and oedometer tests can be simulated by changing the ``strainControl`` mode, and by defining loading controls in axial and radial direction accordingly. A volumetric test can be modelled by setting the axial and radial control functions to the same strain function, whereas an oedometer test runs by setting the radial strain to zero.
+
+
+------------------------------
+Constitutive laws
+------------------------------
+
+Any solid material model implemented in GEOSX can be called by the ``TriaxialDriver``.
+
+For this problem, Extended Drucker Prager model ``ExtendedDruckerPrager`` is used to describe the mechanical behavior of an isotropic material, when subject to external loading.
+As for the material parameters, ``defaultInitialFrictionAngle``, ``defaultResidualFrictionAngle`` and ``defaultCohesion`` denote the initial friction angle, the residual friction angle, and cohesion, respectively, as defined by the Mohr-Coulomb failure envelope.
+As the residual friction angle ``defaultResidualFrictionAngle`` is larger than the initial one ``defaultInitialFrictionAngle``, a strain hardening model is adopted, whose hardening rate is given as ``defaultHardening="0.0001"``. 
+If the residual friction angle is set to be less than the initial one, strain weakening will take place. 
+Setting ``defaultDilationRatio="1.0"`` corresponds to an associated flow rule.
+
+
+.. literalinclude:: ../../../../../inputFiles/solidMechanics/triaxialDriver_ExtendedDruckerPrager.xml
+    :language: xml
+    :start-after: <!-- SPHINX_MATERIAL -->
+    :end-before: <!-- SPHINX_MATERIAL_END -->
+
+
+All constitutive parameters such as density, viscosity, bulk modulus, and shear modulus are specified in the International System of Units.
+
+
+------------------------------------------------------------------
+Functions
+------------------------------------------------------------------
+
+The ``TriaxialDriver`` uses a simple form of time-stepping to advance through the loading steps, allowing for simulating both rate-dependent and rate-independent models.  
+
+In this case, users should define two different time history functions (``strainFunction`` and ``stressFunction``) to describe loading conditions in axial and radial direction respectively. More specifically, the table functions in this example include the temporal variations of radial stress and axial strain, which rely upon the external files in the table directory.
```

Comment:
Link the Function documentation, and also note that simple functions can be embedded directly in the XML.  In fact, in this case, I would recommend it.  The external files are overkill.

Reply:
Add the suggested documentation and note. For the cases with complex loading conditions (such as, combination of triaxial and volumetric compression) , external tables need to be turned on for importing lab controls, which might be frequently used by the users. Therefore, I choose this case to cover this function.